package.json

```diff
@@ -39,33 +39,32 @@
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "^7.1.6",
     "@babel/preset-react": "^7.0.0",
+    "@rollup/plugin-babel": "5.0.0",
```

Comment:
`npm audit` complained about `rollup-plugin-snapshot-size` and upgraded it beyond a version that worked with the current version of `rollup`, so I upgraded Rollup and all its friends.